src/hyperlinks.rst.include

```diff
@@ -6,7 +6,9 @@
 .. _Cylc User Guide: https://cylc.github.io/cylc-doc/latest/html/index.html
 .. _Cylc Workflow Design Guide: https://cylc.github.io/cylc-doc/latest/html/workflow-design-guide/index.html
 .. _Cylc: https://cylc.github.io/
+.. _Cylc 8 Discourse Support Group: https://cylc.discourse.group/c/cylc/7-to-8/13
```

Comment:
Hmm that 13 is strange, although works nicely, just looks a bit odd.

Reply:
Discourse topic number?